README.md

```diff
@@ -1,4 +1,7 @@
 <h5 align="center">
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
```

Comment:
Not rendered accurately 🙂

Reply:
It's true :O I used `all-contributors` CLI to generate this :suspect: . I'm gonna fix it.

Reply:
Fixed!

Reply:
Yea, they have a problem with the badge, you cant have it inline together with other badges either 😅